src/mono/wasm/debugger/DebuggerTestSuite/SetVariableValueTests.cs

```diff
@@ -0,0 +1,138 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.WebAssembly.Diagnostics;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace DebuggerTests
+{
+    public class SetVariableValueTests : DebuggerTestBase
+    {
+        [Theory]
+        [InlineData(1, "a", 10, 30)]
+        [InlineData(1, "a", 10, -1)]
+        [InlineData(1, "b", 20, 30)]
+        [InlineData(2, "c", 30, 60)]
+        [InlineData(3, "d", 50, 70)]
+        public async Task SetVariableValuesAtBreakpointSite(int offset, string variableName, int originalValue, int newValue){
```

Comment:
These method names should mention that they are setting `local` values, and primitive types. Maybe - `SetLocalPrimitiveVariable`?

Reply:
done.